### ISSUE TITLE
Incorrect program window title

### ISSUE THREAD

**Issue by danfe** [CineEncoder/cine-encoder#19]
I've left two comments to the commit 540bc30 which broke it, but they went unanswered, so I guess I have to open an issue instead.  Consider the following patch:
```diff
@@ -11,7 +11,7 @@
    </rect>
   </property>
   <property name="windowTitle">
-   <string>Form</string>
+   <string>Cine Encoder 2021</string>
   </property>
   <property name="windowIcon">
    <iconset resource="files.qrc">
```

**Comment by CineEncoder** [comment]
Hi! I saw the commit 540bc30, but I thought that it was sent by mistake, because there was no comment and I did not find any changes, it seemed to me that just a copy of the project was sent (besides, it would roll back the project to the old version). I accept the WindowTitle change, but since I use FramelessWindowHint, the window title is not visible.

**Comment by danfe** [comment]
> I thought that it was sent by mistake, because there was no comment and I did not find any changes

I see, I guess it's because the commit is so huge that GitHub collapses most parts of it, including the comments; I should have thought of it, sorry.

> I use FramelessWindowHint, the window title is not visible.

I understand, but window title is also displayed in e.g. Alt-Tab window list in most window managers when switching windows.

**Comment by CineEncoder** [comment]
> window title is also displayed in e.g. Alt-Tab window list in most window managers when switching windows.

I've overlooked that. Thanks! I have committed this change.

**Comment by danfe** [comment]
Fixed in 6fd2a0f (unfortunately, together with other, unrelated changes).
